### PR TITLE
Update monitor-prometheus-and-grafana-integration.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 1]
```diff
@@ -50,7 +50,12 @@ To get the scrape_config file for Prometheus, do the following:
 
 ### Step 3. Use Grafana GUI dashboards to visualize the metrics
 
-After your Prometheus service is reading metrics from TiDB Cloud, you can use Grafana GUI dashboards to visualize the metrics.
+After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard json from here and import this json to your own Grafana GUI to visualize the metrics.
```
Comment (ran-huang) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:53]: 加了一个 Grafana import 的链接，看下是否正确~
```suggestion
After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON from here and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/) to visualize the metrics.
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 2]
```diff
@@ -50,7 +50,12 @@ To get the scrape_config file for Prometheus, do the following:
...
+
+    > **Note:**
+    >
+    > Click to download the [Grafana dashboard json](tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json).
```
Comment (ran-huang) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:57]:
```suggestion
    > Click to download the [Grafana dashboard JSON](/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json).
```

Comment (ran-huang) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:57]:
```suggestion
> **Note:**
>
> Click to download the [Grafana dashboard JSON](/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json).
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 3]
```diff
@@ -50,7 +50,12 @@ To get the scrape_config file for Prometheus, do the following:
 
 ### Step 3. Use Grafana GUI dashboards to visualize the metrics
 
-After your Prometheus service is reading metrics from TiDB Cloud, you can use Grafana GUI dashboards to visualize the metrics.
+After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON from here and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/) to visualize the metrics.
+
+> **Note:**
+>
+> Click to download the [Grafana dashboard JSON](/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json).
+
```
Comment (ran-huang) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:58]:
```suggestion
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 4]
```diff
@@ -50,7 +50,11 @@ To get the scrape_config file for Prometheus, do the following:
 
 ### Step 3. Use Grafana GUI dashboards to visualize the metrics
 
-After your Prometheus service is reading metrics from TiDB Cloud, you can use Grafana GUI dashboards to visualize the metrics.
+After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON from here and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/) to visualize the metrics.
+
+> **Note:**
+>
+> Click to download the [Grafana dashboard JSON](/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json).
```
Comment (ran-huang) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:57]:
```suggestion
> Click to download the [Grafana dashboard JSON](https://raw.githubusercontent.com/pingcap/docs/release-6.1/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json).
```

**Comment by lilyjazz** [comment]
@qqqdan PTAL

**Review by qqqdan — APPROVED** [review]

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 5]
```diff
@@ -50,7 +50,11 @@ To get the scrape_config file for Prometheus, do the following:
 
 ### Step 3. Use Grafana GUI dashboards to visualize the metrics
 
-After your Prometheus service is reading metrics from TiDB Cloud, you can use Grafana GUI dashboards to visualize the metrics.
+After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON from here and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/) to visualize the metrics.
+
+> **Note:**
+>
+> Click to get the [Grafana dashboard JSON](https://github.com/pingcap/docs/release-6.1/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json).
```
Comment (ran-huang) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:57]:
```suggestion
After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON from [here](https://github.com/pingcap/docs/release-6.1/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json) and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/) to visualize the metrics.
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json [hunk 6]
```diff
@@ -0,0 +1,1657 @@
...
+      "id": 18,
+      "title": "Query Perfromance",
+      "type": "row"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${DS_PROMETHEUS}"
+      },
+      "description": "database time consumed by SQL statements per second, which are collected by SQL types, such as SELECT, INSERT, and UPDATE.",
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json:79]:
```suggestion
      "description": "database time consumed by SQL statements per second, which is collected by SQL types, such as SELECT, INSERT, and UPDATE.",
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json [hunk 7]
```diff
@@ -0,0 +1,1657 @@
...
+      ],
+      "title": "Databaset Time",
+      "type": "timeseries"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "${DS_PROMETHEUS}"
+      },
+      "description": "The number of SQL statements executed per second in all TiDB instances, which are collected by SQL types, such as SELECT, INSERT, and UPDATE.",
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json:201]:
```suggestion
      "description": "The number of SQL statements executed per second in all TiDB instances, which is collected by SQL types, such as SELECT, INSERT, and UPDATE.",
```

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 8]
```diff
@@ -50,7 +50,7 @@ To get the scrape_config file for Prometheus, do the following:
 
 ### Step 3. Use Grafana GUI dashboards to visualize the metrics
 
-After your Prometheus service is reading metrics from TiDB Cloud, you can use Grafana GUI dashboards to visualize the metrics.
+After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON from [here](https://github.com/pingcap/docs/release-6.1/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json) and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/) to visualize the metrics.
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:53]:
```suggestion
After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON of TiDB Cloud [here](https://github.com/pingcap/docs/release-6.1/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json) and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/) to visualize the metrics.
```

Comment (qiancai) [reply tidb-cloud/monitor-prometheus-and-grafana-integration.md:53]: https://textranch.com/245481/download-from-here/or/download-here/

File: tidb-cloud/monitor-prometheus-and-grafana-integration.md [hunk 9]
```diff
@@ -50,7 +50,7 @@ To get the scrape_config file for Prometheus, do the following:
 
 ### Step 3. Use Grafana GUI dashboards to visualize the metrics
 
-After your Prometheus service is reading metrics from TiDB Cloud, you can use Grafana GUI dashboards to visualize the metrics.
+After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON of TiDB Cloud [here](https://github.com/pingcap/docs/release-6.1/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json) and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/) to visualize the metrics. Then, you can then customize the dashboard as needed by adding or removing panels, changing data sources, and modifying display options.
```
Comment (qiancai) [inline tidb-cloud/monitor-prometheus-and-grafana-integration.md:53]:
```suggestion
After your Prometheus service is reading metrics from TiDB Cloud, you can download the Grafana dashboard JSON of TiDB Cloud [here](https://github.com/pingcap/docs/release-6.1/tidb-cloud/monitor-prometheus-and-grafana-integration-grafana-dashboard-UI.json) and [import this JSON to your own Grafana GUI](https://grafana.com/docs/grafana/v8.5/dashboards/export-import/#import-dashboard) to visualize the metrics. Then, you can then customize the dashboard as needed by adding or removing panels, changing data sources, and modifying display options.
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**